packages/apollos-church-api/src/data/content-items/resolver.js

```diff
@@ -0,0 +1,68 @@
+import { ContentItem as originalContentItem } from '@apollosproject/data-connector-rock';
+import { schemaMerge } from '@apollosproject/server-core';
+// import { get } from 'lodash';
+
+import ApollosConfig from '@apollosproject/config';
+
+const { ROCK_MAPPINGS } = ApollosConfig;
+
+const resolver = {
+  ContentSeriesContentItem: {
+    theme: ({
+      attributeValues: {
+        backgroundColor: { value },
+      },
+    }) => ({
+      type: () => 'LIGHT',
+      colors: () => ({
+        primary: `#${value || 'fff'}`,
+        secondary: '#6bac43',
+        screen: '#F8F7F4',
+        paper: `#fff`,
```

Comment:
Why are these hardcoded and not pulled from config?

Reply:
Good point. Let me see if I can bring these in from `theme`.

Reply:
Doesn't look like the theme module is available within the API since it's based on `UI/KIT`. Pulling it from `UI/KIT` is useless since we override it anyways.

Reply:
No I mean pull from our own `config.yml` or something instead of having hardcoded colors buried in the code. Just some way to make these value external hooks.